index.js

```diff
@@ -53,9 +52,11 @@ const displayBook = ({ title, author, id }, parentElement) => {
   const bookListItemElement = document.createElement('li');

```

Comment:
- [ ] You have done great work, let's improve your code by adding the `display book` function to the Book class as a class method. with this implementation, we make sure that all the methods that have a relation with the Book are defined in the Book class.

Reply:
I disagree with this, the `displayBook` function is used for manipulating the DOM and has nothing to do with the Book class which is  used in place of a book object

Reply:
- Yes I know that the `displayBook` is used for manipulating the DOM, but it still manipulates the information of the book.
```js
displayBook = () => {
  const bookListItemElement = document.createElement('li');
  bookListItemElement.className = 'book-list-item';
  bookListItemElement.innerHTML = `
      <section class="book-store-section display-flex">
        <div class="display-flex">
          <h3>"${this.title}"</h3>&nbsp;
          <span>by</span>&nbsp;
          <p class="paragraph">${this.author}</p>
        </div>
        <button id="${this.id}" type="button" onclick="handleRemove('${this.id}')" class="remove-button">Remove</button>
      </section>`;
  parentElement.appendChild(bookListItemElement);
};
```

Reply:
It **used** the information of the book not manipulate it, the manipulation took place only in the `BookStore` class